databricks_cli/unity_catalog/catalog_cli.py

```diff
@@ -0,0 +1,145 @@
+# Databricks CLI
```

Comment:
include the urls for all the --json-file helps

Reply:
Addressed in #471.